src/libmongoc/src/mongoc/mongoc-client-side-encryption.c

```diff
@@ -1771,6 +1848,11 @@ mongoc_client_encryption_create_key (
 
    ENTRY;
 
+   BSON_ASSERT_PARAM (client_encryption);
+
+   BSON_ASSERT (mongoc_write_concern_get_wmajority (
+      mongoc_collection_get_write_concern (client_encryption->keyvault_coll)));
```

Comment:
I assume this is sanity check. Per the code above, the WC should always be "majority" unless the application tinkers with the private API, correct?